lib/rubocop/sketchup/generator.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module SketchUp
+    class Generator < RuboCop::Cop::Generator
+      def source_path
+        File.join(
+            'lib',
+            'rubocop',
+            'sketchup', # We have our cops nested one extra level.
+            'cop',
+            snake_case(badge.department.to_s),
```

Comment:
Shouldn't this also remove "sketchup_"?

If the command is:

`bundle exec rake new_cop[SketchupRequirements/CopName]`

The path should be:

`lib/rubocop/sketchup/cop/requirements/cop_name.rb`

not:

`lib/rubocop/sketchup/cop/sketchup_requirements/cop_name.rb`

Reply:
Fixed:

```
C:\Users\Thomas\SourceTree\rubocop-sketchup>bundle exec rake new_cop[SketchupRequirements/CopName]
[create] lib/rubocop/sketchup/cop/requirements/cop_name.rb
[create] spec/rubocop/cop/sketchup_requirements/cop_name_spec.rb
[modify] A configuration for the cop is added into config/default.yml.
Do 4 steps:
  1. Modify the description of SketchupRequirements/CopName in config/default.yml
  2. Implement your new cop in the generated file!
  3. Commit your new cop with a message such as
     e.g. "Add new `SketchupRequirements/CopName` cop"
  4. Run `bundle exec rake changelog:new` to generate a changelog entry
     for your new cop.
```